TypeApprovalRegister.yaml

```diff
@@ -209,8 +209,8 @@ paths:
                         subscriber-protocol: 'HTTP'
                         subscriber-address:
                           ip-address:
-                            ipv-4-address: '1.1.3.14'
-                        subscriber-port: 3014
+                            ipv-4-address: '1.1.3.7'
+                        subscriber-port: 3007
               responses:
                 '204':
                   description: 'Subscribed for notifications about approval status updates'
```

Comment:
As per the latest decision, to remove api-segment from http-s/c and tcp-s/c, the uuid's are updated in config and service list files. Similarly can we also update in OAS as well in this PR or shall we create a separate issue for updating the uuid(s) in OAS. Please let me know your views.

Reply:
A separate issue has been already created for this purpose. hence resolving the conversation.